src/rules/requirePrimaryKey.js

```diff
@@ -0,0 +1,35 @@
+export const requirePrimaryKey = {
+  name: 'require-primary-key',
+  docs: {
+    description: 'Enforce primary key definition',
+    url:
+      'https://github.com/kristiandupont/schemalint/tree/master/src/rules#require-primary-key',
+  },
+  process({ options, schemaObject, report }) {
+    const ignorePatternsMatch =
+      options[0] && options[0].ignorePattern
+        ? new RegExp(options[0].ignorePattern)
+        : null;
+    console.log(`IGNORE PATTERN: ${JSON.stringify(options[0].ignorePattern)}`);
+    const validator = ({ tags, columns, name: tableName }) => {
+      const idColumns = columns.filter((c) => c.isPrimary);
+
+      if (idColumns.length < 1) {
+        report({
+          rule: this.name,
+          identifier: `${schemaObject.name}.${tableName}`,
+          message: `The table ${schemaObject.name}.${tableName} does not have a primary key defined";`,
+          suggestedMigration: `ALTER TABLE "${schemaObject.name}.${tableName}" ADD PRIMARY KEY (<primary key column or columns>)";`,
```

Comment:
Hm, this _does_ make sense. However, I sort of picture `suggestedMigrations` as something that could possibly be automatically applied by tooling, which doesn't apply here. Perhaps there should be an additional field for suggestions in English.. We can keep this for now, if you will permit me to change it later on :-)

Reply:
That makes sense, modify as you wish!